# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mrustc/hotfix-cstdint.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mrustc-0.11.2/src/common.hpp.vanilla	2025-09-22 20:32:23.898039792 +0200
+++ mrustc-0.11.2/src/common.hpp	2025-09-22 20:32:30.234000000 +0200
@@ -14,6 +14,7 @@
 #include <set>
 #include <cassert>
 #include <sstream>
+#include <cstdint>
 #include <memory>
 
 #ifdef _MSC_VER
--- mrustc-0.11.2/tools/minicargo/build.h.vanilla	2025-09-22 20:43:58.707241838 +0200
+++ mrustc-0.11.2/tools/minicargo/build.h	2025-09-22 20:44:06.040112124 +0200
@@ -9,6 +9,7 @@
 
 #include "manifest.h"
 #include <path.h>
+#include <cstdint>
 
 class StringList;
 class StringListKV;
